packages/fxa-auth-server/lib/oauth/db/mysql/index.js

```diff
@@ -261,7 +266,7 @@ const QUERY_LIST_ACCESS_TOKENS_BY_UID =
   'FROM tokens LEFT OUTER JOIN clients ON clients.id = tokens.clientId ' +
   'WHERE tokens.userId=?';
 const QUERY_LIST_REFRESH_TOKENS_BY_UID =
-  'SELECT refreshTokens.token AS refreshTokenId, refreshTokens.clientId, refreshTokens.createdAt, refreshTokens.lastUsedAt, ' +
+  'SELECT refreshTokens.token AS tokenId, refreshTokens.clientId, refreshTokens.createdAt, refreshTokens.lastUsedAt, ' +
```

Comment:
Here and elsewhere, I've updating naming to call this the `tokenId` consistently, in line with the change to name it consistently when working with access tokens.